### PR TITLE
[LUA] Convert Magic Numbers [A-E] to Human Readable

### PR DESCRIPTION
**Pull request by N3ckB3ard**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

This PR aims to remove most if not all of the magic numbers found within "\scripts\globals\items"

These PR's will be in pairs of 5 (alphabetically ordered) with as much information I can find regarding any changes in-between

## Notes:

```
"Blaze Feather" effect time is 90 sec
```
http://wiki.ffo.jp/html/6165.html

I see conflicting issues with the amount this item dispenses. I am unsure of the correct reward quantity so I ask for a review of the quantity if you believe it should be changed

**ITEM:** Decennial coat +1, ID: 10253

**DISPENSES:** Moogurt, ID: 5935

```
“When used, gives a random number of Moogurts. (Observed values: 1 to 3.)”
```
https://www.bg-wiki.com/ffxi/Decennial_Coat_%2B1

```
“You can get 1-9 as a stock of Decenial Coat +1 or Decenial Dress +1 .”
```
http://wiki.ffo.jp/html/26963.html

```
Also the link listed from (Rare Ex All ?“Two -”) displays “Moogurt”
```
http://wiki.ffo.jp/html/26981.html

## Steps to test these changes

I have not tested these changes as they are ID lookup updates. If requested I can preform the necessary test of in-game item usage on a local server

### CODE REVIEW

File: scripts/globals/items.lua [hunk 1]
```diff
@@ -1341,6 +1347,7 @@ xi.items =
     LAND_CRAB_MEAT                  = 4400,
     TORTILLA                        = 4408,
     HARD_BOILED_EGG                 = 4409,
+    MELON_JUICE                     = 4424,
```
Comment (claywar) [inline scripts/globals/items.lua]: Can you please ensure that you're using the long name (`name`) for consistency (and tooling)?

For special characters, the following rules apply as well:
```
upName = upName.replace('+', 'P')
upName = upName.replace('_-', '_M')
upName = upName.replace('-', '_')
```

**Review by Xaver-DaRed — APPROVED** [review]

File: scripts/globals/items.lua [hunk 2]
```diff
@@ -1306,6 +1311,7 @@ xi.items =
     MEGALIXIR                       = 4254,
     PINCH_OF_MANA_POWDER            = 4255,
     SLICE_OF_DRAGON_MEAT            = 4272,
+    APPLE_AU_LAIT                   = 4300,
```
Comment (claywar) [inline scripts/globals/items.lua]: `INSERT INTO `item_basic` VALUES (4300,0,'flask_of_apple_au_lait','apple_au_lait',1,1548,58,0,97);`
This should be `FLASK_OF_APPLE_AU_LAIT`

File: scripts/globals/items.lua [hunk 3]
```diff
@@ -1582,12 +1600,18 @@ xi.items =
...
+    DARK_ADAMAN_BULLET_POUCH        = 5873,
     PETRIFY_SCREEN                  = 5876,
     TERROR_SCREEN                   = 5877,
     AMNESIA_SCREEN                  = 5878,
     DOOM_SCREEN                     = 5879,
     POISON_SCREEN                   = 5880,
     OLDE_RARAB_TAIL                 = 5911,
+    ADAMAN_BOLT_QUIVER              = 5913,
+    ADAMAN_BULLET_POUCH             = 5915,
+    DRAGON_FRUIT_AU_LAIT            = 5933,
```
Comment (claywar) [inline scripts/globals/items.lua]: This needs correcting

File: scripts/globals/items.lua [hunk 4]
```diff
@@ -1582,12 +1600,18 @@ xi.items =
...
+    MOOGURT                         = 5935,
```
Comment (claywar) [inline scripts/globals/items.lua]: As does this

**Comment by N3ckB3ard** [comment]
Question:
For mythic items, what would be the best method to introduce the tiers of the mythic weapons in the items.lua? I know some dispense items such as the COR weapon

xi.items.DEATH_PENALTY

What would be the best method to implement this if all Long Names are similar? Or would these best be kept defined within a lua that needs the ID's?

**Comment by TeoTwawki** [comment]
> Question: For mythic items, what would be the best method to introduce the tiers of the mythic weapons in the items.lua? I know some dispense items such as the COR weapon
> 
> xi.items.DEATH_PENALTY
> 
> What would be the best method to implement this if all Long Names are similar? Or would these best be kept defined within a lua that needs the ID's?

we'll need to append a suffix to them.  
DEATH_PENALTY_75
...
DEATH_PENALTY_99
DEATH_PENALTY_99_AFTERGLOW
DEATH_PENALTY_119
DEATH_PENALTY_119_AFTERGLOW

Along those lines I think.

**Review by claywar — APPROVED** [review]

**Pull request merged**